benchmarks/acc2_ns_benchmark.py

```diff
@@ -0,0 +1,163 @@
+import click
```

Comment:
Adding a benchmark for no-slip is unnecessary, since it will have near-identical performance as the original ACC benchmark. Please remove it from this PR (via `git rm`). If you ever need it again, you can find it in the history of your repository. If you don't want to remove it from your master branch, create a separate feature branch and use that for your PR (see e.g. https://gist.github.com/Chaser324/ce0505fbed06b947d962#doing-your-work).